src/sentry/sentry_metrics/indexer/limiters/cardinality.py

```diff
@@ -56,13 +56,12 @@ def _construct_quotas(use_case_id: UseCaseID) -> Optional[Quota]:
             "sentry-metrics.cardinality-limiter.limits.generic-metrics.per-org"
         )
 
-    if quota_args:
-        if len(quota_args) > 1:
-            raise ValueError("multiple quotas are actually unsupported")
+    if not quota_args:
+        raise ValueError("quotas cannot be empty")
```

Comment:
Do we have unit tests for covering this scenario?

Reply:
We have unit tests here that I added which will fail if the quotas is empty, but none that expects an error to be raise when given an empty list. Do you think we should add one?
```
➜  sentry git:(john/disable-empty-card-quota) ✗ pytest tests/sentry/sentry_metrics/test_gen_metrics_multiprocess_steps.py::test_process_messages     
=================================================================================================================================================== test session starts ===================================================================================================================================================
platform darwin -- Python 3.8.16, pytest-7.2.1, pluggy-0.13.1
rootdir: /Users/johnyang/code/sentry, configfile: pyproject.toml
plugins: fail-slow-0.3.0, rerunfailures-11.0, sentry-0.1.11, time-machine-2.13.0, xdist-3.0.2, cov-4.0.0, django-4.4.0
collected 1 item                                                                                                                                                                                                                                                                                                          

tests/sentry/sentry_metrics/test_gen_metrics_multiprocess_steps.py F                                                                                                                                                                                                                                                [100%]

======================================================================================================================================================== FAILURES =========================================================================================================================================================
__________________________________________________________________________________________________________________________________________________ test_process_messages __________________________________________________________________________________________________________________________________________________
tests/sentry/sentry_metrics/test_gen_metrics_multiprocess_steps.py:342: in test_process_messages
    new_batch = MESSAGE_PROCESSOR.process_messages(outer_message=outer_message)
src/sentry/sentry_metrics/consumers/indexer/processing.py:72: in process_messages
    return self._process_messages_impl(outer_message)
src/sentry/sentry_metrics/consumers/indexer/processing.py:113: in _process_messages_impl
    cardinality_limiter_state = cardinality_limiter.check_cardinality_limits(
src/sentry/sentry_metrics/indexer/limiters/cardinality.py:114: in check_cardinality_limits
    configured_quota = _construct_quotas(message["use_case_id"])
src/sentry/utils/metrics.py:199: in inner
    return f(*args, **kwargs)
src/sentry/sentry_metrics/indexer/limiters/cardinality.py:60: in _construct_quotas
    raise ValueError("quotas cannot be empty")
E   ValueError: quotas cannot be empty
================================================================================================================================================= short test summary info =================================================================================================================================================
FAILED tests/sentry/sentry_metrics/test_gen_metrics_multiprocess_steps.py::test_process_messages - ValueError: quotas cannot be empty
==================================================================================================================================================== 1 failed in 4.83s ====================================================================================================================================================
```

Reply:
As long as the error is raised (which appears to be the case here) I think we're good. Thanks!